_data/hosting_providers.json

```diff
@@ -2789,4 +2789,15 @@
     "reviewed": "2024.8.16",
     "note": "This website is in Dutch."
   }
+   ,
+  {
+    "name": "MonoVM",
+    "link": "https://www.monovm.com/",
+    "category": "partial",
+    "tutorial": "https://monovm.com",
```

Comment:
This should ideally be a link to direct instructions for enabling ssl on this product. If that doesn't exist, this can be left blank.